src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/Engine/Trivia/CSharpTriviaFormatter.cs

```diff
@@ -180,32 +180,33 @@ private static bool IsMultilineComment(SyntaxTrivia trivia1)
 
         private bool TryFormatMultiLineCommentTrivia(LineColumn lineColumn, SyntaxTrivia trivia, out SyntaxTrivia result)
         {
-            result = default;
-
-            if (trivia.Kind() != SyntaxKind.MultiLineCommentTrivia)
-            {
-                return false;
-            }
-
-            var indentation = lineColumn.Column;
-            var indentationDelta = indentation - GetExistingIndentation(trivia);
-            if (indentationDelta != 0)
+            if (trivia.Kind() == SyntaxKind.MultiLineCommentTrivia)
             {
-                var multiLineComment = trivia.ToFullString().ReindentStartOfXmlDocumentationComment(
-                    false /* forceIndentation */,
-                    indentation,
-                    indentationDelta,
-                    this.Options.GetOption(FormattingOptions2.UseTabs),
-                    this.Options.GetOption(FormattingOptions2.TabSize),
-                    this.Options.GetOption(FormattingOptions2.NewLine));
-
-                var multilineCommentTrivia = SyntaxFactory.ParseLeadingTrivia(multiLineComment);
-                Contract.ThrowIfFalse(multilineCommentTrivia.Count == 1);
-
-                result = multilineCommentTrivia.ElementAt(0);
-                return true;
+                var indentation = lineColumn.Column;
```

Comment:
view diff with whitespace off.

Reply:
Am I missing something, or did this not need to change though?

Reply:
it did nto.  but when i rome i roman things up.  I personally do not like functions that mix both 'early bail out' and 'nested work'.  I would prefer one or the other.  either do all the early bail out, and do the work at the top level.  Or do nested work and only bail out at the end.

Reply:
All good, just making sure I didn't miss something. The note about viewing the diff just made me think it might have had a little more weight to it.